### DISCUSSION TITLE
Is there a collection of all papers about Koka?

### DISCUSSION THREAD

**Discussion by chtenb** [koka-lang/koka#386] [Q&A] [answered]
I keep finding new ones, and I wonder if there is a collection somewhere, so people can easily browse through all existing material.

**Comment by anfelor** [comment, marked as answer]
Hi @chtenb, [Daan's list of papers](https://www.microsoft.com/en-us/research/people/daan/publications/) should contain all of them! Many papers appear twice on the list: First as a tech report and second as a conference publication. In general, the tech report will be a bit longer and perhaps more up to date, but there should not be significant difference in most cases.
